### PR TITLE
Label Table

### PR DESCRIPTION
**Pull request by danielpeintner**

### CODE REVIEW

**Comment by danielpeintner** [comment]
I tried to look how we can better structure our work for the outside.

grafik

This might give people a _better_ idea.

Having said that, I wonder whether cleaning-up our labels and pointing to https://github.com/w3c/wot-scripting-api/labels is also sufficient (easier to keep in sync).

Labels that are actually used:
* discovery
* enhancement
* F2F ?still needed??
* for next iteration
* Runtime
* Script Management
* use case
* validation

Possible *new* labels:
* ThingDescription

I propose closing labels that are not used (opinions?)
* a11y-needs-resolution
* a11y-tracker
* before publication
* bug
* duplicate
* Editor
* help wanted
* i18n-needs-resolution
* i18n-tracker
* invalid
* privacy-needs-resolution
* privacy-tracker
* propose closing ???
* question
* security-needs-resolution
* security-tracker
* tag-needs-resolution
* tag-tracker
* WD
* wontfix


Additional notes:
* we might want to align capitalization (some are lower-cases, some not)

**Comment by danielpeintner** [comment]
Scripting API Call 2021-08-30

@relu91 keep labels: bug duplicate, help wanted

use GitHub labels instead of table in readme
@danielpeintner will to the changes

**Comment by danielpeintner** [comment]
Note: I updated the PR to simply point to labels.

Moreover, I cleaned-up the labels

removed the following labels (no issue or PR associated):
* a11y-needs-resolution
* a11y-tracker
* before publication
* i18n-needs-resolution
* i18n-tracker
* privacy-needs-resolution
* privacy-tracker
* security-needs-resolution
* security-tracker
* tag-needs-resolution
* tag-tracker
* WD

kept the following issues even though no PR or issue is associated
* bug
* duplicate
* Editor
* help wanted
* invalid
* propose closing
* question
* wontfix

We do have 16 remaining labels which is still a lot.. I guess we can clean-up even more. Any thoughts?

File: README.md [hunk 1]
```diff
@@ -23,3 +23,9 @@ The specification uses WebIDL definitions, but TypeScript definititions are also
 ## Task Force Meeting Logistics
 
 See https://www.w3.org/WoT/IG/wiki/WG_WoT_Scripting_API_WebConf. 
+
+## Technical Matrix
+
+We use [labels](https://github.com/w3c/wot-scripting-api/issues) to categorize our work items.
```
Comment (relu91) [inline README.md:29]: Isn't the wrong link? https://github.com/w3c/wot-scripting-api/labels

**Review by relu91 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielpeintner) [reply README.md]: Yes, I noticed that also :-) 
Thanks!

**Comment by relu91** [comment]
I would remove: 
- help wantend: AFIK outsiders can't really contribute back to the specs. Morevere, usually if we ask for help we invite people to specific issues... so I don't think is really useful. 
- duplicate: I think we should close duplicate issues directly
- invalid: same as duplicate
- wontfix: same

**Review by relu91 — APPROVED** [review]

**Comment by danielpeintner** [comment]
> I would remove:
> ...

👍  down to 12 issues which seems a good number to me

**Comment by zolkis** [comment]
Do we use the "Editor" label?

**Comment by relu91** [comment]
> Do we use the "Editor" label?

I think is for _"Editorial errors"_, it might be useful to keep track of silly mistakes like typos and so on.

**Comment by zolkis** [comment]
Should we call it then "Editorial"? It would be more intuitive. Or is this a standard / generated label?

**Comment by danielpeintner** [comment]
> Should we call it then "Editorial"? It would be more intuitive. Or is this a standard / generated label?

renamed 👍

**Comment by egekorkan** [comment]
I think that everyone has agreed on it, so it can be closed :)

**Comment by danielpeintner** [comment]
Scripting Call 2021-09-06
* keep in mind also https://github.com/w3c/wot-scripting-api/issues/336
* PR looks good to merge

**Pull request merged**